routes/users.js

```diff
@@ -54,13 +54,14 @@ router.patch(
 // upload.single('profile') -> multer inmemory storage of file for type multipart/form-data
 router.post("/picture", authenticate, checkIsVerifiedDiscord, upload.single("profile"), users.postUserPicture);
 router.patch(
-  "/picture/verify/:id",
+  "/picture/verify/:userId",
   authenticate,
   authorizeRoles([SUPERUSER]),
   userValidator.validateImageVerificationQuery,
   users.verifyUserImage
 );
-router.get("/picture/:id", authenticate, authorizeRoles([SUPERUSER]), users.getUserImageForVerification);
+router.get("/picture/all", authenticate, authorizeRoles([SUPERUSER]), users.getAllUsersPhotoVerificationRequests);
```

Comment:
Do we have API contract for this?

Reply:
It is briefly explained in the issue, but not there is not a contract yet, that is to be created.